courses/frontend/react/space-galactica-app/src/components/Footer.module.css

```diff
@@ -2,7 +2,8 @@
   inset-inline-start: 0;
   inset-inline-end: 0;
   inset-block-end: 0;
-  background-color: #eee;
+  position: fixed;
```

Comment:
Instead of setting the Footer to `postion: fixed`, what do you think about making the Layout fill the screen?

Something like this:
```
// main.css
.layout-container {
  display: flex;
  flex-flow: column;
  justify-content: space-between;
  min-height: 100vh;
}
```

Giving the `layout-container` class to the `<div class="inter">` in `Layout.jsx`.

Perhaps this breaks in other scenarios, but I don't really like that the Footer scrolls on top of the main content when the window is smaller 😆